lib/open_mic.rb

```diff
@@ -0,0 +1,36 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+
+  def initialize(info)
```

Comment:
EXCELLENT job catching that this was a single parameter (a hash), and how to dig into the hash for the correct values.